docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>Has the decision been fixed to  publish the docs under <code>documentation/partners/qiskit_ibm_runtime</code>?  At least the library is no longer under the partners organization in Github (https://github.com/qiskit/).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>